internal/tiltfile/tiltfile_state.go

```diff
@@ -1692,6 +1678,53 @@ func toKubernetesApplyCmd(cmd model.Cmd) *v1alpha1.KubernetesApplyCmd {
 	}
 }
 
+func (s *tiltfileState) ignoresForImage(image *dockerImage) (contextIgnores []v1alpha1.IgnoreDef, fileWatchIgnores []v1alpha1.IgnoreDef, err error) {
+	dockerignores, err := s.dockerignoresForImage(image)
+	if err != nil {
+		return nil, nil, fmt.Errorf("reading dockerignore for %s: %v", image.configurationRef.RefFamiliarString(), err)
+	}
+	if image.tiltfilePath != "" {
+		// TODO(milas): technically, this should be a FW ignore, not a context
```

Comment:
fwiw, i think we'd want to do this anyway, so that changes to the tiltfile don't accidentally break the docker cache

Reply:
Ahhh fair, let me remove that TODO